### ISSUE TITLE
Strange Results on TwoRoom and DMC

### ISSUE THREAD

**Issue by SYSU-AIGC** [lucas-maes/le-wm#72]
Thanks for sharing the great work! I am trying to reproduce your results on Huawei's NPUs. On pusht and ogb, the reproduced performance is similar to that reported in the paper. However, on tworoom and dmc, the success rates are much lower (about 20%-40%).  Do you happen to have any insights on what might be causing this？

**Comment by Likeqing2023** [comment]
During my reproduction process in tworoom, I found that using the budget and target steps from the paper, the planning accuracy was only 40%. If the steps are changed to be the same as in the pusht dataset, the average accuracy can reach the level reported in the paper. Could this be an issue with the details in the paper?

**Comment by SYSU-AIGC** [comment]
> During my reproduction process in tworoom, I found that using the budget and target steps from the paper, the planning accuracy was only 40%. If the steps are changed to be the same as in the pusht dataset, the average accuracy can reach the level reported in the paper. Could this be an issue with the details in the paper?

Yes, I downloaded the weights provided by the authors, changed the parameters, and it works. But I still cannot reproduce the result when I try to train the model from scratch.

**Comment by Furina-de-fontaine-queen** [comment]
> During my reproduction process in tworoom, I found that using the budget and target steps from the paper, the planning accuracy was only 40%. If the steps are changed to be the same as in the pusht dataset, the average accuracy can reach the level reported in the paper. Could this be an issue with the details in the paper?

Yes, and under the `TwoRoom` task, training the model for just one epoch yielded better results with a 150 goal offset steps parameter than using the official weights (with a success rate of approximately 54%). However, these results still fell short of those achieved in the paper. It’s possible that this was due to the low dimensionality of the `TwoRoom` data, which didn’t work well with the structural loss function. Further research on `Sub-JEPA` could lead to improvements and optimizations.

**Comment by jarjarvin** [comment]
v3 corrects TwoRoom from 150/100 to 50/25. The 150/100 numbers in Appendix F.1

**Comment by DeLin1001** [comment]
I reproduced and eval the models on A6000 GPUs; and it works perfect in two rooms tasks, achieved 88% sucess rate; but only 60% sucess rate in pusht, whick is much lower than the result showed in the paper; Is there any training setting i should modified?
